NuGet.Docs/Release-Notes/3.5-RC.md

```diff
@@ -0,0 +1,94 @@
+#3.5 RC Release Notes
+
+[Full Changelog]()
+
+[Issues List](https://github.com/NuGet/Home/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%223.5 RC")
+
+**Bugs:**
+
+* Install/restore of a package fails with "Package contains multiple nuspec files." - [#3231](https://github.com/NuGet/Home/issues/3231)
+
+* nuget pack forcefully adds .tt files to content folder no matter what - [#3203](https://github.com/NuGet/Home/issues/3203)
+
+* nuget pack csproj (with project.json) crashes if there are no packOptions and owner in json file - [#3180](https://github.com/NuGet/Home/issues/3180)
+
+* Performance: Improve ContentModel target framework parsing - [#3162](https://github.com/NuGet/Home/issues/3162)
```

Comment:
We should breakout performance updates to a separate section, and provide sample improvment numbers